{"uid":"1990b70fbc34e01d","name":"Checks both cards button shares same color","fullName":"Checks both cards button shares same color","historyId":"c7c298dcfe12e5f54dc4ea0d1e7b28db","time":{"start":1681666562541,"stop":1681666563574,"duration":1033},"status":"failed","statusMessage":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","statusTrace":"TypeError: The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n    at <unknown> (http://localhost:4173/src/bootstrap.tsx:5:10)","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"failed","statusMessage":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","statusTrace":"TypeError: The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n    at <unknown> (http://localhost:4173/src/bootstrap.tsx:5:10)","steps":[{"name":"\"before each\" hook","time":{"start":1681666562541,"stop":1681666562723,"duration":182},"status":"passed","steps":[{"name":"visit http://localhost:4173","time":{"start":1681666562641,"stop":1681666562723,"duration":82},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[],"parameters":[],"stepsCount":1,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true},{"name":"waitUntil {}","time":{"start":1681666562726,"stop":1681666563574,"duration":848},"status":"passed","steps":[{"name":"uncaught exception TypeError: ReactDOM.createRoot is not a function","time":{"start":1681666562727,"stop":1681666563574,"duration":847},"status":"failed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"get button","time":{"start":1681666563573,"stop":1681666563573,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"assert expected **undefined** to have a length of **2** but got **0**","time":{"start":1681666563573,"stop":1681666563573,"duration":0},"status":"failed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666563573,"stop":1681666563573,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666563573,"stop":1681666563573,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666563573,"stop":1681666563573,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666563573,"stop":1681666563573,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666563573,"stop":1681666563573,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666563573,"stop":1681666563573,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666563573,"stop":1681666563573,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666563573,"stop":1681666563573,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666563573,"stop":1681666563573,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666563573,"stop":1681666563573,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666563573,"stop":1681666563573,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666563573,"stop":1681666563573,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666563574,"stop":1681666563574,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666563574,"stop":1681666563574,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"button request","time":{"start":1681666563574,"stop":1681666563574,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[],"parameters":[],"stepsCount":18,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true}],"attachments":[{"uid":"f52243d7cd3a0d00","name":"e2e/tests/runAll.cy.ts:2023-04-16T17:36:03.226Z:attempt-1","source":"f52243d7cd3a0d00.png","type":"image/png","size":155841},{"uid":"3da7591c4f6a0178","name":"video recording","source":"3da7591c4f6a0178.mp4","type":"video/mp4","size":3489871}],"parameters":[],"stepsCount":21,"attachmentsCount":2,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[],"labels":[{"name":"package","value":"e2e.tests.runAll.cy.ts"},{"name":"parentSuite","value":"Vite React Microfrontends"},{"name":"suite","value":"It checks host app"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"739f881766a47331","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681666561725,"stop":1681666562538,"duration":813}}],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":33,"broken":0,"skipped":0,"passed":111,"unknown":0,"total":144},"items":[{"uid":"18e3163a94a28983","reportUrl":"https://module-federation.github.io/module-federation-examples/710//#testresult/18e3163a94a28983","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681596529170,"stop":1681596530806,"duration":1636}},{"uid":"7d94b3643afddda","reportUrl":"https://module-federation.github.io/module-federation-examples/707//#testresult/7d94b3643afddda","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681537401258,"stop":1681537402449,"duration":1191}},{"uid":"c876a873051d6577","reportUrl":"https://module-federation.github.io/module-federation-examples/704//#testresult/c876a873051d6577","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681519372826,"stop":1681519374015,"duration":1189}},{"uid":"7ec99f6bc34481a0","reportUrl":"https://module-federation.github.io/module-federation-examples/696//#testresult/7ec99f6bc34481a0","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681338398763,"stop":1681338400132,"duration":1369}},{"uid":"edabecc14c42b181","reportUrl":"https://module-federation.github.io/module-federation-examples/695//#testresult/edabecc14c42b181","status":"passed","time":{"start":1681324156881,"stop":1681324157937,"duration":1056}},{"uid":"cd0508533a922f1b","reportUrl":"https://module-federation.github.io/module-federation-examples/694//#testresult/cd0508533a922f1b","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681323454564,"stop":1681323456091,"duration":1527}},{"uid":"dece36192bedd9cc","reportUrl":"https://module-federation.github.io/module-federation-examples/693//#testresult/dece36192bedd9cc","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681299805428,"stop":1681299806808,"duration":1380}},{"uid":"d8d5a1c12eb71a9","reportUrl":"https://module-federation.github.io/module-federation-examples/687//#testresult/d8d5a1c12eb71a9","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681214571022,"stop":1681214572691,"duration":1669}},{"uid":"59c7289f16e60f5e","reportUrl":"https://module-federation.github.io/module-federation-examples/686//#testresult/59c7289f16e60f5e","status":"passed","time":{"start":1681146965321,"stop":1681146966981,"duration":1660}},{"uid":"b6549218ef6f14f6","reportUrl":"https://module-federation.github.io/module-federation-examples/685//#testresult/b6549218ef6f14f6","status":"passed","time":{"start":1681146965321,"stop":1681146966981,"duration":1660}},{"uid":"e71d0078f7a324f1","reportUrl":"https://module-federation.github.io/module-federation-examples/684//#testresult/e71d0078f7a324f1","status":"passed","time":{"start":1681136329002,"stop":1681136329970,"duration":968}},{"uid":"71d87e6e993b0935","reportUrl":"https://module-federation.github.io/module-federation-examples/683//#testresult/71d87e6e993b0935","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681099503129,"stop":1681099504378,"duration":1249}},{"uid":"aa238e78fad33946","reportUrl":"https://module-federation.github.io/module-federation-examples/682//#testresult/aa238e78fad33946","status":"passed","time":{"start":1681098137125,"stop":1681098138075,"duration":950}},{"uid":"32db619264ae152b","reportUrl":"https://module-federation.github.io/module-federation-examples/671//#testresult/32db619264ae152b","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680987649443,"stop":1680987650667,"duration":1224}},{"uid":"7b452161b9164630","reportUrl":"https://module-federation.github.io/module-federation-examples/665//#testresult/7b452161b9164630","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680909801349,"stop":1680909802601,"duration":1252}},{"uid":"6ffb36dc3f59611b","reportUrl":"https://module-federation.github.io/module-federation-examples/661//#testresult/6ffb36dc3f59611b","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680834519468,"stop":1680834521528,"duration":2060}},{"uid":"9f9ac9b875f8f7c9","reportUrl":"https://module-federation.github.io/module-federation-examples/657//#testresult/9f9ac9b875f8f7c9","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680806537306,"stop":1680806538687,"duration":1381}},{"uid":"503f652ed96a89ba","reportUrl":"https://module-federation.github.io/module-federation-examples/630//#testresult/503f652ed96a89ba","status":"passed","time":{"start":1680739142309,"stop":1680739143671,"duration":1362}},{"uid":"e7672100ca71c609","reportUrl":"https://module-federation.github.io/module-federation-examples/619//#testresult/e7672100ca71c609","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680701538917,"stop":1680701540501,"duration":1584}},{"uid":"a29908a03ac38e78","reportUrl":"https://module-federation.github.io/module-federation-examples/616//#testresult/a29908a03ac38e78","status":"passed","time":{"start":1680695669548,"stop":1680695670560,"duration":1012}}]},"tags":[]},"source":"1990b70fbc34e01d.json","parameterValues":[]}